### PR TITLE
git checkouts - tiff tzdata zlib zstd

### PR DESCRIPTION
**Pull request by smoser**
convert these packages to git checkout.

### CODE REVIEW

File: tiff.yaml [hunk 1]
```diff
@@ -16,24 +16,28 @@ environment:
       - ca-certificates-bundle
```
Comment (dakaneye) [inline tiff.yaml:16]: we should iterate the epoch

File: tzdata.yaml [hunk 2]
```diff
@@ -16,17 +16,11 @@ environment:
       - wolfi-baselayout
 
 pipeline:
-  - uses: fetch
+  - uses: git-checkout
     with:
-      uri: https://www.iana.org/time-zones/repository/releases/tzcode${{package.version}}.tar.gz
```
Comment (dakaneye) [inline tzdata.yaml:21]: wow, two fetches for stuff in one repo before?

Comment (dakaneye) [reply tzdata.yaml:21]: we should iterate the epoch on tzdata too

**Review by dakaneye — CHANGES_REQUESTED** [review]
LGTM besides epoch updates.

File: tiff.yaml (hunk 1 above)
Comment (smoser) [reply tiff.yaml:16]: will fix epoch.

Comment (smoser) [reply tiff.yaml:16]: fixed.

**Review by dakaneye — APPROVED** [review]

**Pull request merged**